src/utils/analytics.test.ts

```diff
@@ -13,7 +13,7 @@ describe('analytics', () => {
 
   describe('utm', () => {
     it('can set and get utm', () => {
-      global.window = Object.create(window)
+      global.window ??= Object.create(window)
```

Comment:
what is this for?

Reply:
honestly can't remember now, it did fix an issue i was having